src/merge.rs

```diff
@@ -45,6 +71,70 @@ impl MergeValue {
                 hasher.write_byte(*zero_count);
                 hasher.finish()
             }
+            #[cfg(feature = "trie")]
+            MergeValue::ShortCut {
+                key: _,
+                value,
+                height,
+            } => {
+                // try keep hash same with MergeWithZero
+                if value.is_zero() {
+                    return H256::zero();
+                }
+                if *height == 0 {
+                    *value
+                } else {
+                    self.into_merge_with_zero::<H>().hash::<H>()
+                }
+            }
+        }
+    }
+
+    /// Helper function for get base_node
+    /// When call with MergeValue::Value(v), it would be v
+    pub fn base_node<H: Hasher + Default>(&self) -> H256 {
```

Comment:
Where is this function called, or what's the use case?

Reply:
This function is not used anymore, it was used to compare the compability of ShortCut nodes and MergedWithZero nodes's equivalence when developing, and can be removed right now